torch/_weights_only_unpickler.py

```diff
@@ -101,6 +101,7 @@ def _get_allowed_globals():
         torch._utils._rebuild_tensor_v2,
         torch._utils._rebuild_sparse_tensor,
         torch._utils._rebuild_meta_tensor_no_storage,
+        torch._utils._rebuild_nested_tensor,
```

Comment:
Why not add torch._nested_view_from_buffer here directly?

Reply:
I tried that initially but it didn't work? It was getting caught up on the built-in getattr

Reply:
The one for the `torch` module?

Reply:
Yes, I believe that's the one